website/docs/docs/about-dbt-install.md

```diff
@@ -7,21 +7,15 @@ pagination_next: "docs/core/dbt-core-environments"
 pagination_prev: null
 ---
 
-<Constant name="dbt" /> enables data teams to transform data using analytics engineering best practices. You can install dbt locally using these options:
+<Constant name="dbt" /> enables data teams to transform data using analytics engineering best practices. You can choose your local development experience from these tools:
```

Comment:
Minor nit but I think commanding language is helpful since we're flipping here:
```suggestion
<Constant name="dbt" /> enables data teams to transform data using analytics engineering best practices. Choose your local development experience from these tools:
```